packages/logging/src/lib/logger.ts

```diff
@@ -49,21 +56,52 @@ export type LoggerResult = {
   instrumentSocket: (server: Server) => Server
 }
 
-const winstonConsoleFormat = format.combine(
-  format.colorize({ all: true }),
-  format.timestamp(),
-  format.align(),
-  format.printf((info) => `[${info.timestamp}] ${info.level}: ${info.message}`),
-  format.errors({ stack: true })
-)
-
 export const getLogger = ({
   service,
   version,
   level = (process.env.LOG_LEVEL as LogLevel) || 'info',
   showLogs = false, // force show logs on test environments
+  formattingOptions = {
+    colorize: true,
+    timestamp: true,
+    corrId: false,
+    align: true,
+    stack: true,
+  },
 }: LogOptions): LoggerResult => {
+  const defaultFormattingOptions = {
```

Comment:
I dont understand why the defaultFormattingOptions are not the same that formattingOptions parameter passed in the function?

Reply:
In case you pass a partial formattingOptions dict